src/monster.cpp

```diff
@@ -702,10 +703,14 @@ int monster::print_info( const catacurses::window &w, int vStart, int vLines, in
     mvwprintz( w, point( column, vStart++ ), att.second, att.first );
 
     // Awareness indicator in the third line.
-    bool sees_player = sees( get_player_character() );
-    std::string senses_str = sees_player ? _( "Can see to your current location" ) :
+    Character &pc = get_player_character();
+    bool sees_player = sees( pc );
+    const bool player_knows = !pc.has_trait( trait_UNATTENTIVE );
+    std::string senses_str = player_knows && sees_player ? _( "Can see to your current location" ) :
```

Comment:
```suggestion
    std::string senses_str = sees_player ? _( "Can see to your current location" ) :
```

Redundant check. If player_knows is false the result of this assignment is overwritten in the next sentence.